### PR TITLE
Feature/Add Pico support

### PR DESCRIPTION
**Pull request by mikeskydev**
Adds support for the Pico family of devices 🎉
- Adds automatic download and configuration on CI
- New controller model
- Support in build menu

This PR is sponsored by [Pico](https://www.picoxr.com/).

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -150,6 +150,10 @@ jobs:
             targetPlatform: Android
             vrsdk: Oculus
             extraoptions: -setDefaultPlatformTextureFormat astc -btb-il2cpp -btb-experimental
+          - name: Pico
```
Comment (mikeage) [inline .github/workflows/build.yml:153]: Should we call this "Android Pico" to match the other formats?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -150,6 +150,10 @@ jobs:
...
+            targetPlatform: Android
+            vrsdk: Pico
+            extraoptions: -setDefaultPlatformTextureFormat astc -btb-il2cpp
```
Comment (mikeage) [inline .github/workflows/build.yml:156]: Do we also want to do an experimental build?

**Review by mikeage — COMMENTED** [review]
Two comments on the build -- also, should we push the pico APK to itch?

File: .github/workflows/build.yml (hunk 1 above)
Comment (merulaxr) [reply .github/workflows/build.yml:153]: My reasoning was that Pico don't have a desktop headset (yet?) that has a proprietary SDK, so we probably don't need to specify

File: .github/workflows/build.yml (hunk 2 above)
Comment (merulaxr) [reply .github/workflows/build.yml:156]: Sure, though I'm guessing there may not be a huge market for it right now!

**Comment by merulaxr** [comment]
re: itch, I agree but let's wait until Pico are happy with the build and it's out on the store!

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -388,13 +409,20 @@ jobs:
           name: Oculus Quest Experimental
           path: build_oculus_quest_experimental
 
+      - name: Download Build Artifacts (Pico)
```
Comment (mikeage) [inline .github/workflows/build.yml:412]: One last nitpick; if you build pico experimental, it should be added to the release here and in the github release creation. Other than that, looks great!

**Review by mikeage — APPROVED** [review]

**Pull request merged**